services/webnn/ort/graph_builder_ort.h

```diff
@@ -156,11 +156,17 @@ class GraphBuilderOrt {
                       OperandDataType input_data_type,
                       const std::vector<uint32_t>& bias_dims);
 
-  // A helper function used to reorder the weight and bias data for the RNN
+  // A helper function used to transpose the weight or bias layout for the RNN
   // operators (GRU, LSTM, etc.).
+  //
+  // Example:
+  //   To transpose gru weight or bias from "rzn" layout to "rzn" layout, pass
```

Comment:
Nit: "zrn" layout to "rzn" layout (It was my typo in previous comment.
```suggestion
  //   To transpose gru weight or bias from "zrn" layout to "rzn" layout, pass
```